prelude/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+Copyright (c) 2016-2017, Stephen Diehl, 2017, Serokell
```

Comment:
AFAIU it's a completely new package, we can treat it as completely new work that started in 2021. Moreover, I am not sure if we should use MIT here (for consistency with `universum` itself) or MPL-2.0 (which is our default choice for such things). Both are fine to me.

Reply:
Right, somehow missed that the license file can include year/authors too.

Hope for such a small package it does not really matter which license to choose.